website/docs/cloud-docs/api-docs/teams.mdx

```diff
@@ -146,14 +148,14 @@ This POST endpoint requires a JSON object with the following properties as a req
 
 Properties without a default value are required.
 
--> **Note:** You cannot set `manage-workspaces` to `false` when setting `manage-projects` to `true`, since project permissions cascade down to workspaces.
+-> **Note:** You cannot set `manage-workspaces` to `false` when setting `manage-projects` to `true`, since project permissions cascade down to workspaces. This is also the case for `read-workspaces` and `read-projects`, if `read-projects` is `true`, `read-workspaces` must be `true` as well.
```

Comment:
```suggestion
-> **Note:** You cannot set `manage-workspaces` to `false` when setting `manage-projects` to `true`, since project permissions cascade down to workspaces. This is also the case for `read-workspaces` and `read-projects`. If `read-projects` is `true`, `read-workspaces` must be `true` as well.
```

Reply:
I'd break this into two sentences for clarity.